src/_tests_/Navbar.test.js

```diff
@@ -0,0 +1,15 @@
+import { render } from '@testing-library/react';
+import '@testing-library/jest-dom';
+import { BrowserRouter } from 'react-router-dom';
+import Navbar from '../Components/Navbar';
+
+describe('Navbar tests', () => {
+  test('navbar render', () => {
+    const navbar = render(
+      <BrowserRouter>
+        <Navbar />
+      </BrowserRouter>,
+    );
+    expect(navbar).toMatchSnapshot();
+  });
+});
```

Comment:
- [ ] Some of your tests are failing. Kindly fix that

![image](https://user-images.githubusercontent.com/98453979/211101881-b9d69dd1-5522-4d71-b097-4628409c4d2d.png)

Reply:
![Capture](https://user-images.githubusercontent.com/109624418/211132562-44eaea2a-a3cc-48ca-bed7-b173d0c8274c.JPG)

Error in tests has been fixed, now all tests are passing. Please check.
Thank you.